content/js/modal-images.js

```diff
@@ -58,9 +58,9 @@
     </div>`;
 
     var modal = document.getElementById('modalImg');
```

Comment:
I'd recommend using `let` instead of `var` whenever possible.

Reply:
done!